tests/kconfig/arithmetric/testcase.yaml

```diff
@@ -0,0 +1,3 @@
+tests:
+  kconfig.arithmetric:
+    tags: kconfig
```

Comment:
We should limit the amount of test platforms

```yml
    integration_platforms:
      - native_sim
      - native_sim/native/64
```

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/73489#discussion_r1696385076

I applied your suggestion.